spec/puppet-lint/lexer_spec.rb

```diff
@@ -139,6 +139,194 @@
+      expect(tokens[19].value).to eq('$')
+      expect(tokens[19].line).to eq(6)
+      expect(tokens[19].column).to eq(213)
+      expect(tokens[20].type).to eq(:DQMID)
+      expect(tokens[20].value).to eq("(klist | egrep '^Default principal:' | sed 's/Default principal: //') = '")
+      expect(tokens[20].line).to eq(6)
+      expect(tokens[20].column).to eq(215)
+      expect(tokens[21].type).to eq(:VARIABLE)
+      expect(tokens[21].value).to eq('pseudouser')
+      expect(tokens[21].line).to eq(6)
+      expect(tokens[21].column).to eq(289)
+      expect(tokens[22].type).to eq(:DQPOST)
+      expect(tokens[22].value).to eq("'@EXAMPLE.COM")
+      expect(tokens[22].line).to eq(6)
+      expect(tokens[22].column).to eq(301)
+      expect(tokens[23].type).to eq(:COMMA)
+      expect(tokens[23].value).to eq(',')
+      expect(tokens[23].line).to eq(6)
+      expect(tokens[23].column).to eq(315)
+      expect(tokens[24].type).to eq(:NEWLINE)
+      expect(tokens[24].value).to eq("\n")
+      expect(tokens[24].line).to eq(6)
+      expect(tokens[24].column).to eq(316)
+      expect(tokens[25].type).to eq(:INDENT)
+      expect(tokens[25].value).to eq('      ')
+      expect(tokens[25].line).to eq(7)
+      expect(tokens[25].column).to eq(1)
+      expect(tokens[26].type).to eq(:NAME)
+      expect(tokens[26].value).to eq('refreshonly')
+      expect(tokens[26].line).to eq(7)
+      expect(tokens[26].column).to eq(7)
+      expect(tokens[27].type).to eq(:WHITESPACE)
+      expect(tokens[27].value).to eq(' ')
+      expect(tokens[27].line).to eq(7)
+      expect(tokens[27].column).to eq(18)
+      expect(tokens[28].type).to eq(:FARROW)
+      expect(tokens[28].value).to eq('=>')
+      expect(tokens[28].line).to eq(7)
+      expect(tokens[28].column).to eq(19)
+      expect(tokens[29].type).to eq(:WHITESPACE)
+      expect(tokens[29].value).to eq(' ')
+      expect(tokens[29].line).to eq(7)
+      expect(tokens[29].column).to eq(21)
+      expect(tokens[30].type).to eq(:TRUE)
+      expect(tokens[30].value).to eq('true')
+      expect(tokens[30].line).to eq(7)
+      expect(tokens[30].column).to eq(22)
+      expect(tokens[31].type).to eq(:SEMIC)
+      expect(tokens[31].value).to eq(';')
+      expect(tokens[31].line).to eq(7)
+      expect(tokens[31].column).to eq(26)
+      expect(tokens[32].type).to eq(:NEWLINE)
+      expect(tokens[32].value).to eq("\n")
+      expect(tokens[32].line).to eq(7)
+      expect(tokens[32].column).to eq(27)
+      expect(tokens[33].type).to eq(:INDENT)
+      expect(tokens[33].value).to eq('  ')
+      expect(tokens[33].line).to eq(8)
+      expect(tokens[33].column).to eq(1)
+      expect(tokens[34].type).to eq(:RBRACE)
+      expect(tokens[34].value).to eq('}')
+      expect(tokens[34].line).to eq(8)
+      expect(tokens[34].column).to eq(3)
+      expect(tokens[35].type).to eq(:NEWLINE)
+      expect(tokens[35].value).to eq("\n")
+      expect(tokens[35].line).to eq(8)
+      expect(tokens[35].column).to eq(4)
```

Comment:
This is awesome, love the detail! @rodjek just curious, for future maintenance, is there an easier way to lay out the token array for `expect()`, or is this the preferred method?